### PR TITLE
BMI on header and Deceased patient Color, fixed broken tests for changes on ConceptService

### PR DESCRIPTION
**Pull request by larslemos**

### CODE REVIEW

File: app/common/patient/controllers/visitController.js [hunk 1]
```diff
@@ -80,6 +80,13 @@
         );
 
       visitService.search({patient: patientUuid, v: "full"}).then(searchVisitByPatientCallback);
+
+      observationsService.filterLastPatientVitalsObs(patientUuid, Bahmni.Common.Constants.BMI)
+        .then(function (data) {
+          vm.lastBmi = data;
+        }).catch(function (data) {
+
```
Comment (ynurmahomed) [inline app/common/patient/controllers/visitController.js]: Maybe we should notify the user that something went wrong.

File: app/vitals/controllers/vitalsController.js [hunk 2]
```diff
@@ -16,14 +16,18 @@ angular.module('vitals')
...
                 });
 
-                observationsService.get(patientUuid, 'e1e2e934-1d5f-11e0-b929-000c29ad1d07').success(function (data) {
-                    var nonRetired = observationsService.filterRetiredObs(data.results);
-                    $scope.lastHeight = _.maxBy(nonRetired, 'obsDatetime');
-                });
+                observationsService.filterLastPatientVitalsObs(patientUuid, Bahmni.Common.Constants.BMI)
+                  .then(function (data) {
+                        $scope.lastBmi = data;
+                  }).catch(function (data) {
```
Comment (ynurmahomed) [inline app/vitals/controllers/vitalsController.js]: Maybe we should notify the user that something went wrong.

File: app/vitals/views/vitals.html [hunk 3]
```diff
@@ -21,7 +21,7 @@
                 <img src="../../images/bmi.png" class="img-responsive inline-align" width="24" height="24" alt="height">
                 {{'VITALS_LAST_BMI'| translate}}:
                 <span ng-if="lastBmi">
-                    <span class="label label-default">{{lastBmi.value}}</span> {{'COMMON_ON'| translate}} {{lastBmi.obsDatetime| date:'short'}}
+                    <span class="label label-default">{{lastBmi.value}} Kg</span> {{'COMMON_ON'| translate}} {{lastBmi.obsDatetime| date:'short'}}
```
Comment (ynurmahomed) [inline app/vitals/views/vitals.html]: Looks like BMI doesn't have a unit.

File: app/common/patient/directives/patientHeader.html [hunk 4]
```diff
@@ -1,26 +1,30 @@
...
-    <ul class="list-unstyled">
-      <li ng-repeat="identifier in vm.patient.identifiers"
-          ng-class="{'header-text-lg' : identifier.preferred}">
-        {{identifier.identifierType.display}}: <b>{{identifier.identifier}}</b>
-      </li>
-    </ul>
-  </div>
+
+    <p class="control-label"> {{'PATIENT_IS_DEAD' | translate}}
+      </br>{{'PATIENT_INFO_DEATH_DATE' | translate}}: {{vm.patient.deathDate | date: 'medium' }}
```
Comment (ynurmahomed) [inline app/common/patient/directives/patientHeader.html]: Please use one of the date formats defined in: [constants.js](https://github.com/esaude/esaude-emr-poc/blob/master/app/common/application/constants.js). For ```short``` format we don't need to specify.

File: app/common/patient/directives/patientHeader.html [hunk 5]
```diff
@@ -1,26 +1,30 @@
...
+      <div class="header-text-lg">
+        <b>{{vm.patient.fullName}}</b>
+      </div>
+      <div><!-- patient gender, age and phone number -->
+        <span>
+          <img src="../../images/{{vm.patient.gender === 'F' ? 'female.png' : 'male.png'}}"
+               class="img-responsive inline-align" width="24" height="24" alt="male">
+        </span>
+        <span>{{vm.patient.age.years}}</span>
+        <span>{{vm.patient.birthdateEstimated ? '~': ''}}({{vm.patient.birthdate | date:"MM/dd/yyyy"}})</span>
```
Comment (ynurmahomed) [inline app/common/patient/directives/patientHeader.html]: Please use one of the date formats defined in: [constants.js](https://github.com/esaude/esaude-emr-poc/blob/master/app/common/application/constants.js). For short format we don't need to specify.

**Comment by larslemos** [comment]
@ynurmahomed This tests is failing on the `observationsService.filterLastPatientVitalsObs` due to the change of the name on the source to `getLastPatientObs`.

**Comment by ynurmahomed** [comment]
[Calculate BMI](https://trello.com/c/peeCtMND/279-calculate-bmi)

**Comment by ynurmahomed** [comment]
[Provide visual Bar Notification of Deceased](https://trello.com/c/xnPeV1qW/287-provide-visual-bar-notification-of-deceased)

**Comment by larslemos** [comment]
Need to add more tests to Visit Service and ObsService and to improve coverage of VitalsController. Will do it in 1~2h.

**Comment by ynurmahomed** [comment]
[1.0.0_25 Feature: Display vitals header](https://trello.com/c/LPazi5On/194-10025-feature-display-vitals-header)

**Pull request merged**